src/common/transformations/include/transformations/op_conversions/group_query_attention_decomposition.hpp

```diff
@@ -24,15 +24,13 @@ class ov::pass::GroupQueryAttentionDecomposition : public ov::pass::MatcherPass
 
 private:
     ov::OutputVector decompose(std::shared_ptr<ov::op::internal::GroupQueryAttention> node);
+    std::shared_ptr<ov::Node> get_dimensions(const ov::Output<ov::Node>& node, const std::vector<int>& dims);
```

Comment:
Is it required?
Where is implementation of this member?

Reply:
Deleted